pkgs/by-name/ma/managarr/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+  perl,
+}:
+
+let
+  pname = "managarr";
+  version = "0.4.2";
+in
+
```

Comment:
Useless `let` block, please inline those.

Reply:
I can do that, it's not useless though as the version is used twice, and the attribute set isn't recursive (as I thought recursive sets were to be avoided as a general rule of thumb).